src/handlers/cloudBackup.js

```diff
@@ -22,8 +21,13 @@ export const CLOUD_BACKUP_ERRORS = {
   WALLET_BACKUP_STATUS_UPDATE_FAILED: 'Update wallet backup status failed',
 };
 
+export function logoutFromGoogleDrive() {
+  android && RNCloudFs.logout();
+}
+
 // This is used for dev purposes only!
 export async function deleteAllBackups() {
+  await RNCloudFs.loginIfNeeded();
```

Comment:
all the other uses of loginIfNeeded first check if android, not sure if we care to have it here too